### ISSUE TITLE
lint: fix typo in yodaStyleExpr docs

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#663]
From the https://go-critic.github.io/overview#yodaStyleExpr-ref:
```
| yodaStyleExpr | Detects Yoda style expressions that suggest to replace them |
```
Should be "and suggests to replace them".
To fix that, `InitDocumentation` method of the `yodaStyleExprChecker` should be updated.

**Comment by Quasilyte** [comment]
This issue is recommended for https://hacktoberfest.digitalocean.com/.

**Comment by ajcpwnz** [comment]
Opened a PR that fixes this – #664
